bin/scanpy-filter-cells.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import logging
+import scanpy.api as sc
+from scanpy_wrapper_utils import ScanpyArgParser, read_input_object, write_output_object
+
+
+def main(args):
+    logging.debug(args)
+
+    adata = read_input_object(args.input_object_file, args.input_format)
+
+    if args.subset_list:
+        adata = adata[:, adata.obs.isin(args.subset_list)]
+
+    inf, neg_inf = float('Inf'), float('-Inf')
+    for name, high, low in zip(args.parameter_names, args.high_thresholds, args.low_thresholds):
```

Comment:
Are we using these parameter names, high thresholds, low thresholds in other scripts other than filter-cells?

Reply:
It's also used in filter-genes and find-variable-genes.